common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/LastRequestTelemetry.java

```diff
@@ -47,7 +50,7 @@
     }
 
     List<FailedRequest> getFailedRequests() {
```

Comment:
Package-private, right?

Reply:
It is package private

Reply:
Are you looking for some comment indicating that or what?

Reply:
Because the authors left it out of the language, I've always taken it upon myself to say so using comments.

/* package */ void aPackagePrivateFunction()

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1018/commits/89e243e0f75a7d427e781d36a7687773162dd1da